server/infrastructure/supabase/mappers/DepartmentMapper.ts

```
@@ -0,0 +1,33 @@
+      dbDepartment.department_code,
+      dbDepartment.established_department_name
+    );
+  }
+
+  static toPersistence(domain: Department): InsertDepartment {
+    return {
+      department_code: domain.getCode(),
+      established_department_name: domain.getName(),
+    };
+  }
+}
```

Biome (1.9.4)

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)